docsite/docs/api_pages/object-detection.md

```diff
@@ -10,4 +10,424 @@ For more details or to demo it, visit
+         <Text style={styles.permissionText}>
+           React Native Mediapipe needs{' '}
+           <Text style={styles.bold}>Camera permission</Text>.{' '}
+           <Text style={styles.hyperlink} onPress={requestCameraPermission}>
+             Grant
+           </Text>
+         </Text>
+       )}
+     </View>
+   </View>
+ );
+};
+
+
+const styles = StyleSheet.create({
+ root: {
+   flex: 1,
+   justifyContent: 'center',
+   alignItems: 'center',
+   color: 'black',
+ },
+ welcome: {color: 'black', fontSize: 38, fontWeight: 'bold', maxWidth: '80%'},
+ banner: {
+   position: 'absolute',
+   opacity: 0.4,
+   bottom: 0,
+   left: 0,
+ },
+ container: {
+   height: '100%',
+   width: '100%',
+   backgroundColor: 'white',
+   flexDirection: 'column',
+ },
+ camera: {
+   flex: 1,
+ },
+ categoriesText: {color: 'black', fontSize: 36},
+ permissionsContainer: {
+   marginTop: 30,
+ },
+ permissionText: {
+   color: 'black',
+   fontSize: 17,
+ },
+ hyperlink: {
+   color: '#007aff',
+   fontWeight: 'bold',
+ },
+ bold: {
+   fontWeight: 'bold',
+ },
+});
+
+
+export default App;
+```
+
+6. **Reaload app**: and grant acces to camera permissions
+  </TabItem>
+
+  <TabItem value="iOS" label="iOS">
+1. **Give Permissions:** Navigate to your info.plist file in the outermost tag:
+
+```jsx
+// add this at the end of your file before the closing </dict> tag
+
+  <key>NSCameraUsageDescription</key>
+  <string>$(PRODUCT_NAME) needs access to your Camera.</string>
+```
+
+
+2. **Terminal Commands:** In your terminal run the following commands
+
+:::info
+You will only need to run the bundle install command once.
+
+:::
+
+```jsx
+cd ios
+bundle install
+pod install
+```
+
+3. **Locate the configuration file**: from our project and paste it into your project folder. In this case your 'rnmpdemo' folder.
+
+ > If having trouble to find it follow this simple steps:
+ > 1. Navigate to the **native-mediapipe** folder.
+ > 2. Go to the **example** directory.
+ > 3. Enter the **MediapipeExample** directory.
+ > 4. Locate the file named **efficientdet-lite0.tflite**.
+ > 5. Drag and drop or copy and paste the file into the **rnmpdemo** folder.
+
+
+4. **Xcode Launching**:
+
+> 1. Boot **Xcode** on your computer
+> 2. Open existing project, in this case **rnmpdemo**
+> 3. Go to to your **ios** folder
+> 4. Choose the **rnmpdemo.xworkspace**
+> 5. Navigate to your **rnmpdemo** folder inside your **rnmpdemo** directory
+> 6. Add the **efficientdet-lite0.tflite** into your **rnmpdemo** folder
+
+:::warning
+
+Always use the file that ends on '.xworkspace' on your xcode
+
+:::
+
+
+:::info
+
+If you do pod install remember to close xcode, do pod install and restart the app. Because the pod install will change your file.
+
+:::
+
+5. **Run your app**:
+```jsx
+cd ..
+yarn start
+yarn ios
+```
+
+6. **Add Object Detection to your app**: once you see the default React Native welcome page on your device, change your **App.tsx** code to the following one
```

Comment:
This is identical in between ios and android. I'd structure this as the two tabs Android/ios as you have it but keep the platform-independent pieces common, just in order not to confuse people.